ckanext/usmetadata/tests/test_expanded_metadata.py

```diff
@@ -0,0 +1,41 @@
+from formencode import Invalid
+from ckanext.usmetadata.plugin import expanded_metadata
+
+
+def validate(sid, values):
+    """ test accrual_periodicity values """
+
+    for em in expanded_metadata:
+        if em['id'] == sid:
+            validators = em['validators']
+
+    for test in values:
+        for validator in validators:
+            # if fails will raise an error
+            print "validating {} for {} at {}".format(test, sid, validator)
+            try:
+                validator.to_python(test)
+                print " - OK {} for {} at {}".format(test, sid, validator)
+            except AttributeError:  # it's a simple function validator
+                pass  # extra validation functions
+            except Invalid:
+                print " - INVALID {} for {} at {}".format(test, sid, validator)
+                raise
+
+
+def test_accrual_periodicity():
+    tests = [
+        'Annual',
+        'monthly',
+        'P3Y6M4DT12H30M5S',
+        'P1Y2DT3M',
+        'P1Y2D']
```

Comment:
So instead of `P3Y6M4DT12H30M5S` we should have `R/P3Y6M4DT12H30M5S`, for `P1Y2DT3M` `R/P1Y2DT3M` etc.